core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -960,18 +963,18 @@ public void completeFile(AlluxioURI path, CompleteFileOptions options)
    * Writes to the journal.
    *
    * @param inodePath the {@link LockedInodePath} to complete
+   * @param flushCounter counter used for journaling operations
    * @param options the method options
-   * @return the flush counter for journaling
    * @throws InvalidPathException if an invalid path is encountered
    * @throws FileDoesNotExistException if the file does not exist
    * @throws BlockInfoException if a block information exception is encountered
    * @throws FileAlreadyCompletedException if the file is already completed
    * @throws InvalidFileSizeException if an invalid file size is encountered
    */
-  private long completeFileAndJournal(LockedInodePath inodePath, CompleteFileOptions options)
+  private void completeFileAndJournal(LockedInodePath inodePath, Mutable<Long> flushCounter,
```

Comment:
possible to always have `flushCounter` as the last argument as a convention?
I noticed that for some `xxxAndJournal` method (like `completeFileAndJournal`, `flushCounter` is before `options`, while others (like `deleteAndJournal`), the counter is after. Either way will work for me, just keep it consistent.